src/app/rpmostree-builtin-status.c

```diff
@@ -268,17 +300,10 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
             }
         }
 
+      if (origin_requested_packages)
+        print_packages ("RequestedPackages", max_key_len, origin_requested_packages);
       if (origin_packages)
-        {
-          g_autofree char *packages_joined = NULL;
-          g_autoptr(GPtrArray) origin_packages_sorted = g_ptr_array_new ();
-          for (char **iter = (char**) origin_packages; iter && *iter; iter++)
-            g_ptr_array_add (origin_packages_sorted, *iter);
-          g_ptr_array_sort (origin_packages_sorted, rpmostree_ptrarray_sort_compare_strings);
-          g_ptr_array_add (origin_packages_sorted, NULL);
-          packages_joined = g_strjoinv (" ", (char**)origin_packages_sorted->pdata);
-          print_kv ("Packages", max_key_len, packages_joined);
-        }
+        print_packages ("LayeredPackages", max_key_len, origin_packages);
```

Comment:
I haven't tried this locally yet but I feel like it's going to be noisy with more than a few packages.  E.g. on my workstation today:

```
● atomic-ws:atomicws/fedora/x86_64/continuous
             Version: 25.2017.40 (2017-02-22 02:32:22)
          BaseCommit: 1709567188619732ce67a7fc0e92123a4ee73b4cc419c26e0b32c3358e3bbde7
              Commit: 4cce8da42ce3b184cf11c378f927aaabea41aaf5691eb368dcbced902509d75d
              OSName: fedora
            Packages: ansible emacs fuse-sshfs gdb keepassx krb5-workstation libgit2 libvirt opensc pcsc-lite-ccid powerline strace vagrant-hostmanager vagrant-libvirt vagrant-sshfs virt-manager xchat xsel ykclient ykpers
```

Doubling that wouldn't be the end of the world...but maybe we could do a diff here?  I.e. treating `LayeredPackages` and `RequestedPackages` as sets, we print `RequestedPackages - LayeredPackages`?